.github/workflows/release.yml

```diff
@@ -1,11 +1,15 @@
 name: Release
 
 on:
+  pull_request:
+    types: [closed]
+    branches:
+      - master
   workflow_dispatch:
     inputs:
       version:
         description: 'Release version'
-        required: true
+        required: false
```

Comment:
Let's make it required to not allow trigger this workflow manually without specified version.